db/migrate/20231118153157_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
+      t.integer :comment_counter
+      t.integer :likes_counter
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Kindly add reference to the user table in the post table to reference the user and post relationship in the ERD diagram. Example snippet:

```ruby
   create_table :posts do |t|
    t.string :title
    t.text :text
    t.integer :comments_counter
    t.integer :likes_counter
    t.timestamps # this will crate uopdated_at and created_at columns
    t.references :author, foreign_key: { to_table: 'users' } # this will create author_id and a corresponding index  (but students can also add column and index in separate commands)
  end



```

Reply:
well resolved! ✔️